iosApp/Survey/Sources/Presentation/Views/EmbedLoadingView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  EmbedLoadingView.swift
+//  Survey
+//
+//  Created by Bliss on 27/9/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct EmbedLoadingView<Content: View>: View {
```

Comment:
How about using `ViewModifier` in this case?

I read this library: https://github.com/globulus/swiftui-generic-dialog/blob/main/Sources/SwiftUIGenericDialog/SwiftUIGenericDialog.swift

So for loading, we can have the same approach

Reply:
I added the approach you suggested, looks nice. https://github.com/blyscuit/kmm-swiftUI-IC/pull/56/commits/d42d493ec1a297624b8965dde89ca09c4db30730